package.json

```diff
@@ -53,11 +53,11 @@
     "format:js": "prettier --write '**/*.js'",
     "format:md": "prettier --write --parser markdown '**/*.md'",
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
-    "test": "jest --config jest.test.config.js",
-    "test:watch": "jest --config jest.test.config.js --watch",
-    "test:bundle": "jest --config jest.bundle.config.js",
+    "test": "cross-env NODE_ICU_DATA=node_modules/full-icu jest --config jest.test.config.js",
```

Comment:
Sadly npm hooks don't support wildcards (e.g. `pretest*`) but maybe a `setup` run script is nice to ease this and then just have `yarn setup && jest ...`.

Reply:
Hm... can you clarify what `setup` would look like? 😄

Reply:
`"setup": "cross-env NODE_ICU_DATA=node_modules/full-icu"` :D

Reply:
I tried this and it didn't work 🤔 ? In the test I put yarn setup && jest ...

Reply:
Ah spawned in different shell maybe.